src/main/java/com/taskviewer/api/model/jwt/JwtRs.java

```diff
@@ -0,0 +1,14 @@
+package com.taskviewer.api.model.jwt;
+
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.Setter;
+
+@Getter
+@Setter
+@AllArgsConstructor
+public class JwtRs {
+
+  private JwtToken accessToken;
+  private JwtToken refreshToken;
```

Comment:
I don't tolerate compound var names

you can simplify your code:
JwtToken is a type of your var, so name can be `access` and `refresh`